packages/odyssey-react/src/components/Heading/Heading.module.scss

```diff
@@ -12,58 +12,58 @@
 
 .root {
   margin-block-start: 0;
-  margin-block-end: $spacing-xs-em;
+  margin-block-end: var(--ContentMarginBlockEnd);
```

Comment:
why the component part prefix here for `Content` ? Seems like we could avoid that entirely for Heading as its so simple